mesh2hrtf/Mesh2Input/mesh2input.py

```diff
@@ -200,6 +209,7 @@ def draw(self, context):
         row.prop(self, "programPath")
         row = layout.row()
         row.prop(self, "pictures")
+        row.prop(self, "checkall")  #Added by Sanket
```

Comment:
In case we add separate parameters for different checks, this should get its own subcategory